packages/@o3r/localization/schematics/localization-base/index.ts

```diff
@@ -139,7 +139,7 @@ export function updateLocalization(options: { projectName?: string | null | unde
         workspaceProject.architect.build.options?.assets?.map((a: { glob: string; input: string; output: string }) => a.output).find((output: string) => output === '/localizations');
 
       if (!alreadyExistingBuildOption) {
-        workspaceProject.architect.build.options.assets.push({
+        (workspaceProject.architect.build.options.assets || []).push({
```

Comment:
I think it might be less confusing to just add the condition in the `if` instead of creating a useless array that does nothing
```typescript
      if (!alreadyExistingBuildOption && workspaceProject.architect.build.options.assets) {
```